packages/ui-kit/src/components/Counter/Counter.types.ts

```diff
@@ -33,6 +33,8 @@ export type CounterQueryProps = {
 
   /** This prop allows you to override the URL for Propel's GraphQL API. You shouldn't need to set this unless you are testing. */
   propelApiUrl?: string
+  /** Indicates specific time zone region */
+  timeZone?: string
```

Comment:
Can we please reduce the prop duplicates by following https://github.com/propeldata/ui-kit/pull/141#discussion_r1423838991?